File: python3/1074.py
entrada = int(input())

for index in range(0, entrada):
    teste = int(input())
    if(teste > 0):
        if(teste % 2 == 0):
            print("EVEN POSITIVE")
        else:
            print("ODD POSITIVE")
    elif(teste < 0):
        if(teste % 2 == 0):
            print("EVEN NEGATIVE")
        else:
            print("ODD NEGATIVE")
    else:
        print("NULL")

File: cpp/beginner/1074.cpp
#include <iostream>

using namespace std;

int main() {

    int n, i, j;

    cin >> n;

    for(i = 1; i <= n; i++) {
        cin >> j;
        if(j == 0) {
            cout << "NULL" << endl;
        } else {
            if (j % 2 == 0) {
                if (j > 0) {
                    cout << "EVEN POSITIVE" << endl;
                }
                if (j < 0) {
                    cout << "EVEN NEGATIVE" << endl;
                }
            }
            if (j % 2 != 0) {
                if (j > 0) {
                    cout << "ODD POSITIVE" << endl;
                }
                if (j < 0) {
                    cout << "ODD NEGATIVE" << endl;
                }
            }
        }
    }

    return 0;
}
